docs/manage/fleet/reuse-configuration.md

```diff
@@ -33,33 +33,28 @@ You must be an [organization owner](/manage/manage/rbac/) to create fragments fo
 
 {{< table >}}
 {{% tablestep link="/operate/get-started/setup/" %}}
-**1. Configure one machine**
+**1. In the [Viam app](https://app.viam.com), create a machine** in any {{< glossary_tooltip term_id="location" text="location" >}}.
 
-Start by configuring one of your machines.
+{{% /tablestep %}}
+{{% tablestep link="/operate/get-started/setup/" %}}
+**2. Use the CONFIGURE tab to configure all resources** you want to use on all your machines.
```

Comment:
```suggestion
**2. Use the CONFIGURE tab to configure all the resources** you want to use on your machines.
```
Otherwise sounds almost like you're configuring them on each of your machines at this step, and reads oddly